api.swagger.yaml

```diff
@@ -1791,6 +1765,12 @@ paths:
                     properties:
                       result:
                         $ref: '#/components/schemas/TopicResponse'
+          headers:
+            Location:
+              schema:
+                type: string
+                format: uri
+              description: The location at which the new resource can be found
```

Comment:
When you say location do you mean the URI of the API endpoint to GET the resource from?

Reply:
Yep so in this example it would have a `Location: /api/v0/topics/0` header or something like that

Reply:
Cool